SpringData/src/main/java/com/epam/jmp2/controller/AccidentDataController.java

```diff
@@ -0,0 +1,46 @@
+package com.epam.jmp2.controller;
+
+import java.text.ParseException;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.epam.jmp2.entities.Accident;
+import com.epam.jmp2.model.RoadAccident;
+import com.epam.jmp2.service.AccidentService;
+
+@RestController
+public class AccidentDataController {
+	
+	@Autowired
+	AccidentService accidentService;
+	
+	@RequestMapping(value = "/findOneByIndex/{indexId}",  method = RequestMethod.GET)
```

Comment:
maybe a better naming would be more suitable, like

/find/byIndex/{index}
/find/byRoadCondition/{roadCondition}